### ISSUE TITLE
letters typed by ibus-hangul are deleted when they are selected.

### ISSUE THREAD

**Issue by DarthGandalf** [kvirc/KVIrc#1260]
**Reported by crocket on 27 Feb 2012 14:34:28 UTC**
Many korean users use ibus-hangul to type korean letters.
When I select a block of korean letters typed by ibus-hangul, by pressing Shift + Arrow keys,
those letters are deleted one by one.

Although japanese characters typed by ibus-anthy are not deleted like that, since xchat has no problem with ibus-hangul, the bug may reside in KVIrc.

---

Migrated from: https://svn.kvirc.de/kvirc/ticket/1260

**Comment by DarthGandalf** [comment]
**Modified by HelLViS69 on 02 Jan 2015 14:50:01**
Version: 4.0.4 Insomnia → 4.2.0 Equilibrium

**Comment by un1versal** [comment]
lovely example of how migrating svn bug reports is less than ideal, now how the heck do we go about contacting user or whatever to get feedback or ask him/her to test with recent build :D?

**Comment by DarthGandalf** [comment]
@un1versal you want to go back to SVN? :)

**Comment by un1versal** [comment]
eh, no :) though technically never was there ;). Im just not sure what to do with these bug reports.

**Comment by ghost** [comment]
Confirmed, still exists in latest git pull.

**Comment by ghost** [comment]
I'm assuming it's somewhere in `KviInputEditor::previousCharSelection()` and the repaint/update when selection has changed.

This probably works on XChat because they're likely using a Qt input widget rather than a custom QWidget like KVIrc is using.

**Comment by pragmaware** [comment]
@TheReign : can you still reproduce it with the latest git?

**Comment by ghost** [comment]
yes

**Comment by un1versal** [comment]
This issue is related to https://github.com/kvirc/KVIrc/issues/1882 perhaps same bug triggered another way

**Comment by un1versal** [comment]
Yes, after went into #1882 I found that IBus cased that bug so IMO the two are connected even if the way they are triggered is different.

**Comment by un1versal** [comment]
is this reproducible with newer Qt? seems too close to call on relation with 1882 and @TheReign is saying nothing on it.
